openapi.yaml

```
@@ -0,0 +1,528 @@
+openapi: 3.1.0
+info:
+  title: Codex API
+  description: |
+    Authentication API for the Codex application.
+    
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)